config/purposes/bespoke_hic.wip.yml

```diff
@@ -0,0 +1,3 @@
+---
+LBB Lib PCR-XP Norm:
```

Comment:
Just checking you are happy this plate will have all the default labware creator, presenter, size 96, plate barcode label printing, concentration files etc. options.
I think it's ok, straight stamp creation, it is a 96-well plate. Maybe ask them to check ok at UAT.

Reply:
Yeah I'm going to ask them to test it first.